packages/file-explorer/src/FileStorage.ts

```diff
@@ -43,6 +43,7 @@ export interface FileStorageTable extends StorageTable<FileStorageItem> {
    * @param expanded What expanded state to set
    */
   setExpanded(path: string, expanded: boolean): Promise<void>;
+  collapseAll(): Promise<void>;
```

Comment:
Add JSDocs to the method to explain how it's used.
Also you'll need to add the `breaking` label to this PR, and mention in the PR description there is a breaking change now that `collapseAll()` is added, so all implementations of `FileStorageTable` will need to be updated.

Another option would instead to not change the API at all, and instead use the existing `setExpanded` method to have collapseAll behaviour, e.g. `setExpanded('/', false)`. Though I guess existing implementations would still need to handle that special case as well, so it could introduce more subtle bugs.

I'll mention it in our 1:1